src/library/scala/collection/mutable/Map.scala

```diff
@@ -122,6 +122,8 @@ trait MapOps[K, V, +CC[X, Y] <: MapOps[X, Y, CC, _], +C <: MapOps[K, V, CC, C]]
     r
   }
 
+  override def clear(): Unit = { keysIterator foreach -= }
```

Comment:
I think we should remove that override since it’s the same as the base implementation, right?

Reply:
Yep good call